src/state/lists/hooks.ts

```diff
@@ -132,29 +132,36 @@ export function combineMaps(map1: TokenAddressMap, map2: TokenAddressMap): Token
   }, {}) as TokenAddressMap
 }
 
+export function combineMapss(maps: TokenAddressMap[]): TokenAddressMap | null {
+  if (maps.length < 2) return maps.length ? maps[0] : null
+  const chainIdSet = new Set()
+  maps.forEach(map => Object.keys(map).forEach(chainId => chainIdSet.add(chainId)))
+  const chainIds: ChainId[] = Array.from(chainIdSet).map(Number)
+
+  return chainIds.reduce<Mutable<TokenAddressMap>>((memo, chainId) => {
+    memo[chainId] = {}
+    maps.reverse().forEach(map => Object.assign(memo[chainId], map[chainId]))
+    return memo
+  }, {}) as TokenAddressMap
+}
+
 // merge tokens contained within lists from urls
 function useCombinedTokenMapFromUrls(urls: string[] | undefined): TokenAddressMap {
   const lists = useAllLists()
+  const filteredUrls = urls
```

Comment:
Why not useMemo here to remove the stringify below?

Reply:
why add Memo here can remove stringify below?